src/game/client/draw/w3dmodeldraw.cpp

```diff
@@ -1154,3 +1178,1892 @@ void W3DModelDrawModuleData::Parse_Condition_State(INI *ini, void *instance, voi
         data->m_conditionStates.push_back(info);
     }
 }
+
+W3DModelDraw::W3DModelDraw(Thing *thing, ModuleData const *module_data) :
+    DrawModule(thing, module_data),
+    m_animMode(RenderObjClass::ANIM_MODE_LOOP),
+    m_isDaytime(true),
+    m_curState(nullptr),
+    m_hexColor(0),
+    m_renderObject(nullptr),
+    m_shadow(nullptr),
+    m_shadowState(true),
+    m_decalShadow(nullptr),
+    m_trackRenderObject(nullptr),
+    m_whichAnimInCurState(-1),
+    m_nextState(nullptr),
+    m_loopDuration(-1),
+    m_recalcBones(false),
+    m_fullyObscuredByShroud(false)

```

Comment:
`m_pauseAnimation` seems not initialized.

Reply:
fixed.

Reply:
m_needRecalcBoneParticleSystems missing too

Reply:
I see no member by that name anywhere in w3dmodeldraw.h or w3dmodeldraw.cpp.